packages/blockchain-extension/extension/commands/deleteEnvironmentCommand.ts

```diff
@@ -73,9 +76,23 @@ export async function deleteEnvironment(environment: FabricEnvironmentTreeItem |
                 await vscode.commands.executeCommand(ExtensionCommands.DISCONNECT_GATEWAY);
             }
 
+            if (_environment.environmentType === EnvironmentType.LOCAL_ENVIRONMENT) {
+                delete localSettings[_environment.name];
+
+                try {
+                    await vscode.commands.executeCommand(ExtensionCommands.TEARDOWN_FABRIC, undefined, true, _environment.name);
+                } catch (err) {
+                    // Ignore
```

Comment:
we should probably give a warning message here, so that if it doesn't tear down properly then they may need to clean up their docker environment

Reply:
Updated to log message - and added a test.